modules/trusted-firmware-m/tfm_boards/CMakeLists.txt

```diff
@@ -187,3 +195,9 @@ endif()
 
 file(COPY ${CMAKE_CURRENT_LIST_DIR}/common
      DESTINATION ${INSTALL_PLATFORM_NS_DIR})
+
+
+if((NRF_SOC_VARIANT STREQUAL nrf54l15) OR (target STREQUAL nrf54l15))
```

Comment:
This is probably not the best, I'm open to suggestions for how to best make this file available for NS.